components/use-store.js

```diff
@@ -1,7 +1,7 @@
 import create from 'zustand'
 
 const url =
-  'https://raw.githubusercontent.com/carbonplan/compliance-users/main/data/outputs/user_data_v3.0.json'
+  'https://raw.githubusercontent.com/carbonplan/compliance-users/main/data/outputs/user_data_v4.0.json'
```

Comment:
I temporarily pointed this at `https://raw.githubusercontent.com/carbonplan/compliance-users/89408702a5c3c2e9b67fae954fb6b03acd4b4152/data/outputs/user_data_v4.0.json` and it looked good locally to me!

Reply:
Thanks, Kata! I had forgotten about that pattern. (cc @badgley for previewing the the new data on vercel)

I just changed the date in `zenodo.js` and had already changed the dates in `layout.js`, so I think we're good on that front.